uiowa/uiowa
# Change Scholarships Views Query

Resolves #2568

Client expects that checking all the type/resident checkboxes, in the case of https://admissions.prod.drupal.uiowa.edu/scholarship/collegiate-and-departmental-scholarships should display that scholarship on each of their configured views.

This basically means that the views query should be switched to ORs instead of ANDs and that there are no duplicates with these results.

I removed the heavy adjustment to get ANDs to work and confirmed that the ORs were working as expected similar to the current site:

- http://admissions.local.drupal.uiowa.edu/first-year-resident-scholarships === /finances/scholarships-nonres-first-year-students
- http://admissions.local.drupal.uiowa.edu/first-year-non-resident-scholarships === /finances/scholarships-first-year-students
- http://admissions.local.drupal.uiowa.edu/transfer-student-scholarships (has a duplicate view in the page) === /finances/scholarships-transfer-students
- http://admissions.local.drupal.uiowa.edu/international-student-scholarships === /finances/scholarships-international-first-year-students

# To Test

- `blt ds --site=admissions.uiowa.edu`
- Check the above current site -> new site views for a match in results. If it doesn't look quite right confirm that the scholarship has been created and tagged the same way on the new site.
